### PR TITLE
Add travelling to Pycon page and update the corresponding links on other pages

### PR DESCRIPTION
**Pull request by aswanipranjal**
This issue adds details about **how to get to PyCon** (travelling to pycon) page and updates the link of that page on other HTML pages.

Addresses #91

### CODE REVIEW

**Comment by aswanipranjal** [comment]
@pythonindia/pycon-2018-team @ananyo2012 @vipulgupta2048 @realslimshanky please review?

File: travel-2018.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+  </nav>
+
+
+  <!-- Section: about -->
+
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
```
Comment (vipulgupta2048) [inline travel-2018.html]: Remove extra space

File: travel-2018.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+					    <p><b>Beware of taxi soliciting tours at the airport greeting area; they might try to scam you into exorbitant rates (especially so in case of non- locals)</b>. The Hyderabad Traffic Police has partnered with the Airport authorities to run a counter for prepaid taxis. This is a safe option with all taxis being registered with the police. The police also runs a SHE cab service now, especially for women travellers, with female drivers and safety equipment such as a GPS connected to the Police Control Room.</p>
+
+					    <p>One can also hire a cab from many app- based services such as Uber or Ola.
+					    <u><a href="https://book.olacabs.com/?pickup_name=Rajiv%20Gandhi%20International%20Airport%20(RGIA)%2C%20Shamshabad%20Hyderabad%20Telangana&lat=17.24066&lng=78.428747&drop_lat=17.4622103&drop_lng=78.3568492&drop_name=Kondapur%2C%20Hyderabad%20Telangana%20India">Here is a fair estimate using OLA.</a></u></p>
```
Comment (vipulgupta2048) [inline travel-2018.html]: I think adding a link to booking an Ola is not needed, Hence can be removed.
#suggestion

File: travel-2018.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+					    <p>If 3-4 of you are travelling together, then it is possible to directly take a cab as it will be cheaper (Please keep in mind that while these services are substantially cheaper than the radio cabs, they will charge Rs. 200 in addition to the fare as parking charges levied on them by the airport). This option is the best while travelling to the airport, as the parking charges do not apply for departures.</p>
+
+					    <p>Hired cars are also available from a booth just before walking outside of the airport. This gives you the advantage of paying in advance, thereby avoiding any disagreements over price.The airport can be contacted on their (toll free for BSNL/MTNL subscribers) number 1 800 419-2008 for all services and inquiries including arrivals / departure information, facilities, transport availability, etc.</p>
```
Comment (vipulgupta2048) [inline travel-2018.html]: price.The airport > price. The airport

File: travel-2018.html [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
+        <div class="pycon-travel">
+        	<p><u><a href="https://wikitravel.org/en/Hyderabad">Here is a handy Wiki page</a></u> describing about the various ways you can travel to Hyderabad. The main points have been highlighted below.</p>
+       		<ul>
+       			<li>
+              <h3>Via Air</h3>
+       				<p><b>To Hyderabad: </b>Domestic connectivity to Hyderabad is excellent with Indian airlines operating from here including Air India, Air India Express, Indian Airlines, Indigo Airlines, Jet Airways, JetLite, Trujet and SpiceJet. (Kingfisher is now defunct, and Trujet is new).</p>
```
Comment (vipulgupta2048) [inline travel-2018.html]: Can you add information about international connectivity, or if there are more than one terminals. Which gate would be the most feasible to exit out of.

Comment (aswanipranjal) [reply travel-2018.html]: RGIA has combined international and domestic airports. Apart from that I could not find any information on which would be the most appropriate gate to exit out of.

**Comment by aswanipranjal** [comment]
I've tried to address all the issues that @vipulgupta2048 pointed out. I created a new pull request using a branch from my fork as it is the right way. I am closing this PR and the discussion can continue on #182.

**Pull request closed without merging**